assets/css/case-studies.scss

```
@@ -2287,7 +2613,10 @@ html:has(.cs-sticky-nav) {
       margin-bottom: 0.15rem;
       color: #8aa09f;
     }
+    // __after is a flex row — force its mobile tag onto its own line above
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)